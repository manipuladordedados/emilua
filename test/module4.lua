-- leaf_cannot_import_child

require('module4_foo')
