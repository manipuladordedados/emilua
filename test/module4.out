Main fiber from VM 0x1 panicked: 'A leaf module cannot import child modules'
stack traceback:
	[string "?"]: in function 'require'
	input.lua:3: in main chunk
	[C]: in function ''
	[string "?"]: in function <[string "?"]:0>
